backend/src/apiserver/metadata/metadata_store_test.go

```diff
@@ -0,0 +1,123 @@
+package metadata
+
+import (
+	"fmt"
+	mlpb "ml_metadata/proto/metadata_store_go_proto"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
```

Comment:
please format import, same in other files

Reply:
nit - some gofmt/goimport tools don't remove empty lines for you. you might want to manually remove them to keep consistent.

Reply:
Gofmt should just auto-run on save, so I'd rather not mess with it :-) It'll just introduce inconsistencies. The blank line is correct here, in that it tries to separate external and internal packages. However, ml_metadata is a little messed up, in that there is no proper go package for it (you can't do go get on it). Hence, the reason it looks like this. I'll look into getting this fixed upstream. For now though, I think gofmt is doing the right thing.